nuget/bblanchon.PDFium.Binaries.Android.nuspec

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>bblanchon.PDFium.Binaries.Android</id>
+    <version>$version$</version>
+    <authors>Benoit Blanchon</authors>
+    <owners>Benoit Blanchon</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="expression">Apache-2.0</license>
+    <licenseUrl>https://licenses.nuget.org/Apache-2.0</licenseUrl>
+    <projectUrl>https://github.com/bblanchon/pdfium-binaries</projectUrl>
+    <description>This package contains pre-compiled binaries of the PDFium library, an open-source library for PDF manipulation and rendering.</description>
+    <releaseNotes>$releaseNotes$</releaseNotes>
+    <copyright>Copyright © Benoit Blanchon 2017-2022</copyright>
+    <tags>PDFium PDF binaries library native Chromium</tags>
+    <repository type="git" url="https://github.com/bblanchon/pdfium-binaries.git" branch="$branch$" commit="$commit$" />
+  </metadata>
+
+  <files>
+    <!-- the native PDFium binaries -->
+    <!-- see list of available runtime identifiers -->
+    <!-- https://docs.microsoft.com/en-us/dotnet/core/rid-catalog -->
+    <file src="pdfium-android-arm/lib/libpdfium.so" target="runtimes/android-arm/native/libpdfium.so" />
```

Comment:
`runtimes/android-arm/native/libpdfium.so`: is this path somehow imposed?
Could we do `runtimes/android/arm/libpdfium.so`?

Reply:
This is the convention for NuGet packages: `runtimes/$platform$-$architecture$/native/yourlib.so`

Otherwise .NET won't be able to extract the package properly.